### ISSUE TITLE
Always redirecting to codeyourcloud.com/about when two google accounts are configured

### ISSUE THREAD

**Issue by terala** [mkaminsky11/codeyourcloud#9]
Sign in with two active google accounts, with one of them linked to drive. Open a supported text file from the authorized drive. It launches codeyourcloud.com and redirects to codeyourcloud.com/about and stays there. When one of the google accounts is signed out, this app starts to work again.

I think there is an issue with the code with multiple google accounts. Can this be fixed?

**Comment by mkaminsky11** [comment]
This is a known issue, but I have not figured out a way to resolve it yet.

**Comment by binwiederhier** [comment]
Any workarounds? This is still an issue!

**Comment by mkaminsky11** [comment]
No, I'm still trying to figure it out. It is a problem with Google's authorization API

**Comment by fredghostkyle** [comment]
:+1:
